File: fortran/lab/armstrong.f
        write(*, *) 'Enter Number'
        read(*, *) num

        itemp=num
        isum=0
 10     if(num .eq. 0) goto 20
               irem = mod(num, 10)
               isum = isum + irem**3
               num = num/10
               goto 10
 20     if(isum .eq. itemp) goto 5
               write(*, *) 'NOT ArmStrong Number'
        stop
 5      write(*, *) 'ArmStrong Number'
        stop
        end

File: c/misc/armstrong.c
/* Armstrong Number from 1 to 1000  */
#include<stdio.h>
int main()
{
   int r,sum=0,num,temp;
   for(num=1;num<=1000;num++)
   {
      temp=num;
      while(temp>0)
      {
	 r=temp%10;
	 sum=sum+r*r*r;
	 temp=temp/10;
      }
      if(sum==num)
      {
	printf("%d \t",num);
      }
      sum=0;
   }
   getch();
   return 0;
}
